sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultPodSpec.java

```diff
@@ -111,16 +123,17 @@ public Integer getCount() {
     }
 
     @Override
-    public List<TaskSpec> getTasks() {
-        return tasks;
-    }
+    public List<TaskSpec> getTasks() { return tasks; }
 
     @Override
     public Optional<PlacementRule> getPlacementRule() {
         return Optional.ofNullable(placementRule);
     }
 
     @Override
+    public Optional<Collection<NetworkSpec>> getNetworks() { return Optional.ofNullable(networkSpecs); }
```

Comment:
With collections, we tend not to wrap in `Optional` and instead let something like `Collection.emptyList` represent the null case. That way, you don't need to branch on non-emptiness, and instead can just always iterate.